Documentation/ApiOverview/Internationalization/Index.rst

```diff
@@ -16,6 +16,6 @@ Internationalization and localization
 .. toctree::
    :titlesonly:
 
-   Introduction
-   ManagingTranslations
-   XliffFormat
+   Introduction/Index
```

Comment:
Are you sure that this is necessary? Maybe @marble can have a look at this.

Reply:
I would say this change was not necessary, because there are other folders all around that keep the files under the same root folder, without using "Index.rst" filenames and that's not a problem. I just wanted to keep an homogeneous structure.
In any case, if you wanna keep this commit, I fixed the includes path at the beginning with my last commig.
If not, I can revert all back.

Reply:
@mbrodala Should I revert this for the merge request to be aproved, or we wait for marble's answer?

Reply:
My recommendation is to not use the `index` file with a folder on top.
Instead keep the filesystem structure short and use `XliffFormat.rst` instead of `XliffFormat/Index.rst`.

So I would suggest to revert this.

Reply:
It does not matter much - I'll keep what is currently proposed.

Reply:
Ah, now I got it. Nah - this changes the document structure online again, so to not have trouble with links / google at this place, i'd rather keep the explicit file namings - I'll thus push a commit for this again.
BTW: I like it to not always have the sub directories with only a single file. Imho we should only use sub dirs if we're gathering more stuff in there.